<?xml version="1.0" encoding="utf-8"?>
<launch>

    <arg name="map" default="map2"/>

    <!-- ROS STAGE -->
    <param name="use_sim_time" value="true"/>
    <param name="map_dir" value="$(find autonomous_exploration)/sim_worlds/$(arg map).png"/>
    <node pkg="stage_ros" type="stageros" name="stageros" args="-d $(find autonomous_exploration)/sim_worlds/$(arg map).world" output="screen"/>  


    

</launch>
 
